pipelines/migration/projeto_subsidio_sppo/constants.py

```diff
@@ -709,3 +710,289 @@ class constants(Enum):  # pylint: disable=c0103
             },
         },
     }
+
+    SUBSIDIO_SPPO_PRE_CHECKS_LIST = {
+        "sppo_realocacao": {
+            "check_gps_capture__sppo_realocacao": {
+                "description": "Todos os dados de realocação foram capturados"
+            }
+        },
+        "sppo_registros": {
+            "check_gps_capture__sppo_registros": {
+                "description": "Todos os dados de GPS foram capturados"
+            }
+        },
+        "gps_sppo": {
+            "check_gps_treatment__gps_sppo": {
+                "description": "Todos os dados de GPS foram devidamente tratados"
+            }
+        },
+        "sppo_veiculo_dia": {
+            "dbt_utils.unique_combination_of_columns__data_id_veiculo__sppo_veiculo_dia": {
+                "description": "Todos os dados de status dos veículos foram devidamente tratados"
```

Comment:
```suggestion
                "description": "Todos os registros são únicos"
```

Alterar para todos os casos semelhantes